<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" tests="6" skipped="0" failures="0" errors="0" timestamp="2021-05-09T12:40:37" hostname="wangweilidembp" time="0.711">
  <properties/>
  <testcase name="clearFail()" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.259"/>
  <testcase name="getFailPutExceptionFail()" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.109"/>
  <testcase name="evictFail()" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.076"/>
  <testcase name="putFail()" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.081"/>
  <testcase name="getFail()" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.099"/>
  <testcase name="getPutNewElementFail()" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.085"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
